auto-notify.plugin.zsh

```diff
@@ -107,7 +107,8 @@ function _auto_notify_message() {
           -e 'on run argv' \
           -e 'display notification (item 1 of argv) with title (item 2 of argv)' \
           -e 'end run' \
-          "$body" "$title"
+          "$body" "$title" \
+          2> >(grep -Ev 'ApplePersistence=(NO|YES)' >&2)
```

Comment:
could you add a code comment explaining this? What you put in the PR description would work